python/xorbits/_mars/dataframe/statistics/tests/test_statistics_execution.py

```diff
@@ -147,22 +144,26 @@ def test_dataframe_corr(setup):
 
     df = DataFrame(raw)
 
-    result = df.corr()
```

Comment:
Better have `numeric_only=False` covered by some test cases. 

I think we can create an issue for this for now and fix it later. What do you think?

Reply:
`numeric_only=True` is the default behavior before 2.0, these changes just keep tests behavior, add cases in later PR if needed.

Reply:
> `numeric_only=True` is the default behavior before 2.0, these changes just keep tests behavior, we can add cases in later PR if needed.

Yeah I have read the release notes. Could you create some issue for the tests later?

Reply:
Sure.